src/WebJobs.Script/Description/FunctionMetadata.cs

```diff
@@ -36,23 +36,6 @@ public FunctionMetadata()
 
         public ScriptType ScriptType { get; set; }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether the function is disabled.
-        /// <remarks>
-        /// A disabled function is still compiled and loaded into the host, but it will not
-        /// be triggered automatically, and is not publicly addressable (except via admin invoke requests).
-        /// </remarks>
-        /// </summary>
-        public bool IsDisabled { get; set; }
```

Comment:
In the v1.x branch IsDisabled allowed Http trigger functions to be disabled. Since the core SDK work you did relies on simply not starting the listener for disabled functions. However, that doesn't work for http. This used to be handed here: https://github.com/Azure/azure-webjobs-sdk-script/blob/v1.x/src/WebJobs.Script.WebHost/FunctionRequestInvoker.cs#L109, though it looks like that code wasn't carried forward in the Core branch?

Reply:
All this likely means that we need to retain this IsDisabled property - the thing that changes is how the value is computed.